routes/person.js

```diff
@@ -8,7 +8,7 @@ const sortRelated = require('../lib/sort-related-items');
 module.exports = (elastic, config) => ({
   method: 'GET',
   path: '/people/{id}/{slug?}',
-  handler: (request, reply) => HTMLResponse(request, reply, elastic, config),
+  handler: (request, reply) => reply.continue(),
```

Comment:
Yes let's wait, I need more thoughts on how to manage the content negociation with the user agent